reference/forms/types/checkbox.rst

```diff
@@ -5,8 +5,8 @@ checkbox Field Type
 ===================
 
 Creates a single input checkbox. This should always be used for a field that
-has a Boolean value: if the box is checked, the field will be set to true,
-if the box is unchecked, the value will be set to false.
+has a ``Boolean`` value: if the box is checked, the field will be set to ``true``,
```

Comment:
it should not be in backticks here. It is about the English adjective in this sentence